### ISSUE TITLE
Update the install section with 3.9 after October 1st

### ISSUE THREAD

**Issue by pushfoo** [pvcraven/arcade_book#25]
TL;DR: The "Installing Python" section uses 3.8, which'll [EOL when Python 3.13 releases in October 2024](https://devguide.python.org/versions/#supported-versions)

### On the site / Steps To Replicate

1. Go to https://learn.arcade.academy/en/latest/chapters/02_setup/setup.html
2. Scroll "Installing Python" (or [click here](https://learn.arcade.academy/en/latest/chapters/02_setup/setup.html#installing-python))
3. Observe the page shows 3.8.X in two screnshots:
 
    1. In the image showing how to add Python to PATH ![image](https://github.com/user-attachments/assets/fe069953-7ffc-4151-92aa-0552362d98c4)
    2. In the subsequent image: 
![image](https://github.com/user-attachments/assets/650f5a62-ef15-46af-b9f0-f8d1dc8f706b)

### In the code

The lines which include these images are here:
https://github.com/pvcraven/arcade_book/blob/b82e980779964b0b35daf3f0712ece7c656ad900/source/chapters/02_setup/setup.rst?plain=1#L100-L107
